composer.json

```diff
@@ -70,7 +70,7 @@
     },
     "conflict": {
         "doctrine/annotations": "<1.12",
-        "sensio/framework-extra-bundle": "<5.2.3",
+        "sensio/framework-extra-bundle": "<6.1",
```

Comment:
I had to drop support for older versions since attributes support for the `Template` annotation was added in https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/707

Reply:
:+1: